molgenis-biobank-plots/src/main/frontend/biobank-plots/src/store/state.js

```diff
@@ -0,0 +1,20 @@
+export const state = {
+  numberOfSamples: 0,
+  rnaseq: false,
+  DNAm: false,
+  DNA: false,
+  wbcc: false,
+  metabolomics: false,
+  male: false,
+  female: false,
+  smoking: false,
+  nonSmoking: false,
+  biobank: null,
+  aggs: [],
+  biobanks: [],
+  attributeCharts: [],
+  server: {
+    apiUrl: 'https://molgenis09.gcc.rug.nl/api/'
```

Comment:
Since your requests originate from the same domain you can probably use relative URL here.